ddpui/tests/integration_tests/dbt_automation/test_bigquery_ops.py

```
@@ -27,32 +31,140 @@
+
+
+def scaffold(config, _warehouse, tmpdir):
+    """Create a dbt project for integration tests and write a bigquery profile."""
+    project_name = config["project_name"]
+    default_schema = config["default_schema"]
+    project_root = Path(tmpdir)
+    dbt_bin = shutil.which("dbt")
+    if not dbt_bin:
+        raise RuntimeError("dbt binary is not available on PATH")
+
+    subprocess.check_call(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)